roles/core/cluster/tasks/cluster.yml

```diff
@@ -74,6 +74,35 @@
     - scale_cluster_profile_name is defined and scale_cluster_profile_name != 'None'
     - scale_cluster_profile_name not in gpfs_cluster_system_profile
 
+- block:
+    - name: check | cluster profile name validation
+      assert:
+        that:
+          - scale_cluster_profile_name is not match("gpfs.*")
+        msg: >-
+          A user-defined profile must not begin with the string 'gpfs'
+
+    - name: check | cluster profile format validation
+      assert:
+        that:
+          - stat_user_profile_result.matched == 1
+        msg: >-
+          A user-defined profile must have the .profile suffix
+  delegate_to: localhost
+  when:
+    - scale_cluster_profile_name is defined and scale_cluster_profile_name != 'None'
+    - scale_cluster_profile_name not in gpfs_cluster_system_profile
```

Comment:
Same comment as the test on line 66  above,   `- name: check | Find GPFS user profile` 

I think this case is wrong, because of the comment at line 81.  We are using `scale_cluster_profile_name` for both system and user defined profiles, however, line 81 does not let us specify a system defined profile.